tests/StarkEx.Crypto.SDK.DifferentialTests/Signing/StarkExSignerDifferentialTests.cs

```diff
@@ -0,0 +1,60 @@
+namespace StarkEx.Crypto.SDK.DifferentialTests.Signing;
+
+using FluentAssertions;
+using Org.BouncyCastle.Math;
+using StarkEx.Crypto.SDK.DifferentialTests.Helpers;
+using StarkEx.Crypto.SDK.DifferentialTests.Helpers.Attributes;
+using StarkEx.Crypto.SDK.Signing;
+using Xunit;
+
+public class StarkExSignerDifferentialTests
+{
+    [Theory]
+    [Repeat(10)]
+    public void SignMessage_InputsAreValid_ResultsMatch(int runId)
+    {
+        // Generate random inputs.
+        var messageHash = RandomHelpers.GetRandomBigInteger(new BigInteger("2").Pow(251)).ToString(16);
+        var privateKey = RandomHelpers.GetRandomBigInteger(new BigInteger("2").Pow(251)).ToString(16);
+
+        // Run C# implementation.
+        var target = CreateStarkExSigner();
+        var result = target.SignMessage(messageHash, privateKey);
+
+        // Run Python implementation.
+        var controlResult = PythonHelpers.Sign(messageHash, privateKey);
+
+        // Compare results.
+        result.R.Should().Be(controlResult.R);
+        result.S.Should().Be(controlResult.S);
+    }
+
+    [Theory]
+    [Repeat(10)]
+    public void VerifySignature_DifferentialTesting_ResultsMatch(int runId)
+    {
+        // Generate random inputs.
+        var messageHash = RandomHelpers.GetRandomBigInteger(new BigInteger("2").Pow(251)).ToString(16);
+        var privateKey = RandomHelpers.GetRandomBigInteger(new BigInteger("2").Pow(251)).ToString(16);
+
+        // Compute other inputs.
+        var target = CreateStarkExSigner();
```

Comment:
Why is this needed when target is not used and the method is static? Can this be deleted?

Reply:
I'm not sure of I follow here, what method are you referring to?

Reply:
The variable `target` has no uses, and since the method `CreateStarkExSigner` is static, there are no side effects besides the output (and eventually on the input for reference variables, but it is not the case here)